jprov/server/http.go

```diff
@@ -53,10 +53,18 @@ func checkVersion(cmd *cobra.Command, w http.ResponseWriter, ctx *utils.Context)
 		ctx.Logger.Error(err.Error())
 	}
 
+	// clientCtx, error := client.GetClientTxContext(cmd)
+	// if error != nil {
+	// 	ctx.Logger.Error(err.Error())
+	// 	return
+	// }
+	// chainID := clientCtx.ChainID
+
 	var v types.VersionResponse
 	if len(res) > 0 {
 		v = types.VersionResponse{
 			Version: res,
+			ChainID: "test-chain",
```

Comment:
This hard coded ChainID isn't even showing up, I believe it has something to do with the logic:

if len(res) > 0 

The default version flag is actually empty, and if the provider doesn't enter a version flag, it will stay as empty so the response will be 0, which means the block in the if statement won't run.